test/lib/ReaderFilter.js

```diff
@@ -0,0 +1,66 @@
+const test = require("ava");
+const sinon = require("sinon");
+const ReaderFilter = require("../../lib/ReaderFilter");
+
+test("_byGlob: Basic filter", async (t) => {
+	const abstractReader = {
+		_byGlob: sinon.stub().returns(Promise.resolve(["resource a", "resource b"]))
```

Comment:
```suggestion
		_byGlob: sinon.stub().returns(Promise.resolve(["Resource A", "Resource B"]))
```

Reply:
Thank you for the feedback! However, as this is test code (identifiable by the leading `test/` segment in the file's path) and since the string is a stand-in for an actual object, I won't apply this suggestion.